cob_moveit_bringup/bin/add_robot.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
cob_moveit_config/add_robot.sh and cob_moveit_bringup/add_robot.sh are the same and both work...